### PR TITLE
Add NXP_MIMXRT1060_EVK target to README

### PR DESCRIPTION
**Pull request by MateuszKlatecki**
Added new port NXP_MIMXRT1060_EVK in README.md

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Signed-off-by: GITHUB_USER <GITHUB_USER_EMAIL>

### CODE REVIEW

**Comment by MateuszKlatecki** [comment]
Fixed

File: targets/FreeRTOS/NXP/include/platform_target_capabilities.h [hunk 1]
```diff
@@ -16,7 +16,16 @@ extern "C" {
 
 // this platform doesn't have any declared capabilities
 
-// the targets of this platform don't have any declared capabilities
+// the targets of this platform can declate these capabilities
```
Comment (josesimoes) [inline targets/FreeRTOS/NXP/include/platform_target_capabilities.h]: Does the NXP have DFU update? If not, this enum isn't doing nothing here and should be removed.

Comment (MateuszKlatecki) [reply targets/FreeRTOS/NXP/include/platform_target_capabilities.h]: iMX RT have something similar to DFU. NXP has a MFGTool that allows you to program Flash memory via USB. Additionally, this eval board support OpenSDA MSD. Thanks to it, after connecting usb, a disk appears, which is enough to copy the file with the new firmware. If you used to play Mbed, it works here almost identically.

Comment (josesimoes) [reply targets/FreeRTOS/NXP/include/platform_target_capabilities.h]: Nice! So this can be of use. Let's switch to Discord channel to discuss this.

Comment (josesimoes) [reply targets/FreeRTOS/NXP/include/platform_target_capabilities.h]: Please remove this from the PR, as it's supposed to on the README and we are good to go.
The enum should go with another PR (atomicity principle 😉 ).

**Pull request merged**